theSrc/scripts/PlotlyChartElements.js

```diff
@@ -82,7 +82,7 @@ function createPlotlyData (config) {
         }
     } else {
         const indices_by_group = _.groupBy(indices, i => config.group[i])
-        const group_names = Object.keys(indices_by_group)
+        const group_names = _.uniq(config.group)
```

Comment:
What does this change do? Is `Object.keys` removing some values?

Reply:
It changes the order of `group_names` for an edge case where the groups are numbers "1","1.1", etc. Somehow the previous order was "1","2","3","1.1","1.2",... whereas now it is in the order in which it appears in `config.group`.